src/modules/lighthouse/showReports.ts

```diff
@@ -0,0 +1,44 @@
+import ora from 'ora'
+import chalk from 'chalk'
+
+import log from '../../logger'
+import { lighthouse } from '../../clients'
+
+import { TableGenerator } from './TableGenerator'
+
+function allWhenUndefined(atribute: string): string {
+  return !atribute ? '<all>' : atribute
+}
+
+function appUrlFormatString(app: string, url: string) {
+  return `${chalk.green(allWhenUndefined(app))} and url: ${chalk.blue(allWhenUndefined(url))}`
+}
+
+export async function showReports(app: string, url: string) {
+  if (!app && !url) {
+    log.error('You must specify, at least, app or url flags to query reports')
+    return
+  }
+
+  const spinner = ora(
+    // eslint-disable-next-line prettier/prettier
```

Comment:
Why this was necessary?

Reply:
This can be removed and then running `yarn format` will fix this

Reply:
It is not! I changed the formatted string, the first one was bigger and the linter suggested that, but then I changed and forgot to remove that. Thanks!